src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -178,8 +205,19 @@ public Object invokeMethod(String name, Object args) {
 
     /**
      * When {@link #invokeMethod(String, Object)} is calling a {@link StepDescriptor}
+     * @deprecated Prefer {@link #invokeStep(StepDescriptor, String, Object)}
```

Comment:
Noticed while resolving merge conflicts in #211 that this is still being called for metasteps like `core` or `wrap`, in which case the `functionName` is not very useful. Amending that to pass the symbol, though really tracking ambiguity there would require more work.